### ISSUE TITLE
getInUserLists() error + documentation missing

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/wishlist#95]
**Description**
I want to check if an item is in any list (across list types too).
I just found the, undocumented, `getInUserLists()` through #88, but I think there's a bug?
`Undefined variable: forUser`, the `$forUser` and `$forceSave` variables are not known to that method.

If I delete that code, it seems to work fine.

**Comment by engram-design** [comment]
This function is certainly under-done, sorry! Should be fixed for the next release. To get this fix early, change your `verbb/wishlist` requirement in `composer.json` to:

```
"require": {
  "verbb/wishlist": "dev-craft-3 as 1.4.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.11](https://github.com/verbb/wishlist/releases/tag/1.4.11)
